### PR TITLE
Copy update on /download/iot/intel-iotg 

### LINKED ISSUE #10930: [COPY UPDATE] /download/iot/intel-iotg - "Download Ubuntu for Intel IoT platforms"
Please update /download/iot/intel-iotg - "Download Ubuntu for Intel IoT platforms" from the [copy doc](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit?usp=drivesdk)

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Updated link URLs

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Be sure to test on mobile, tablet and desktop screen sizes
- Please check page against [copy doc](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit) and resolve comments

## Issue / Card

Fixes #10930 

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

**Comment by minkyngkm** [comment]
Currently waiting for a link to be provided by PM

**Review by sowasred2012 — COMMENTED** [review]
All looks good so far - and I've added the changed links as suggestions in the copy doc :+1:

File: templates/download/iot/intel-iotg.html [hunk 1]
```diff
@@ -53,14 +53,14 @@ <h3>Ubuntu Core 20</h3>
...
         <a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/focal/release/inteliotg/ubuntu-core-20-amd64+intel-iot.img.xz">Download</a>
       </p>
-      <p>Please see the <a href="https://assets.ubuntu.com/v1/377806cd-October_25-Release+Notes.pdf">release notes</a> and <a href="https://assets.ubuntu.com/v1/87e46aa7-Installation+Instructions+Core+Image+Intel+IoTG+Platforms.pdf">instructions</a> for more information.</p>
+      <p>Please see the <a href="https://assets.ubuntu.com/v1/ab850656-November_24-Release+Notes.pdf">release notes</a> and <a href="https://assets.ubuntu.com/v1/87e46aa7-Installation+Instructions+Core+Image+Intel+IoTG+Platforms.pdf">instructions</a> for more information.</p>
     </div>
     <div class="col-6 p-divider__block">
       <h3>Ubuntu Desktop 20.04</h3>
       <p>
-        <a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/focal/release/inteliotg/ubuntu-20.04-preinstalled-desktop-amd64+intel-iot.img.xz">Download</a>
+        <a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/focal/release/inteliotg/focal-desktop-amd64+intel-iot-20211110.5.iso">Download</a>
```
Comment (sowasred2012) [inline templates/download/iot/intel-iotg.html:61]: We got the link earlier this evening ([copy doc](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit?disco=AAAASXJyg8Q))
```suggestion
        <a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/focal/release/inteliotg/ubuntu-20.04-desktop-amd64+intel-iot.iso">Download</a>
```

**Comment by minkyngkm** [comment]
Thanks @sowasred2012 for reviewing! Its been all updated now :)

**Comment by darkolarczyk** [comment]
@minkyngkm could you please let me know once this has been updated? It looks like everything is ready on our side :)

**Review by sowasred2012 — APPROVED** [review]
LGTM!

**Comment by darkolarczyk** [comment]
@minkyngkm and @sowasred2012 I don't think the links to the release notes have been changed? I still see the links to the october release notes.

**Comment by sowasred2012** [comment]
@darkolarczyk we've just pushed those changes live, you should see them on the site now.

**Pull request merged**